beamer/tests/contracts/test_fill_manager.py

```diff
@@ -1,132 +1,136 @@
-import brownie
-
-from brownie import FillManager, TestL2Messenger
+import ape
+from ape import project
+from hexbytes import HexBytes
 
 from beamer.tests.agent.utils import make_address
 from beamer.tests.util import alloc_accounts, alloc_whitelisted_accounts, create_request_id
 
 
-def test_fill_request(fill_manager, token):
-    chain_id = brownie.web3.eth.chain_id
+def test_fill_request(fill_manager, token, accounts):
+    chain_id = ape.chain.chain_id
     amount = 100
     (receiver,) = alloc_accounts(1)
-    (filler,) = alloc_whitelisted_accounts(1, {fill_manager})
+    (filler,) = alloc_whitelisted_accounts(1, [fill_manager])
 
-    with brownie.reverts("Ownable: caller is not the owner"):
-        fill_manager.addAllowedLp(filler, {"from": receiver})
+    with ape.reverts("Ownable: caller is not the owner"):
+        fill_manager.addAllowedLp(filler, sender=receiver)
 
     whitelist_tx = fill_manager.addAllowedLp(receiver)
-    assert "LpAdded" in whitelist_tx.events
+    assert [x for x in fill_manager.LpAdded.from_receipt(whitelist_tx)]
```

Comment:
I remember we talked about this also with ape people. Do we have an issue on ape side for making it easier to check for events?

Reply:
In cases like these it is better to use `any`:

```python
assert any(fill_manager.LpAdded.from_receipt(whitelist_tx))
```
because you are not creating a whole list and iterating through the entire sequence.

Reply:
i think there is .events now but maybe not like we are using before i may need to actually constract it somehow like assert LpAdded(....) in whitelist_tx.events

Reply:
events changed basically and now ok i guess